code/game/gamemodes/nuclear/nuclearbomb.dm

```diff
@@ -407,7 +407,7 @@ var/bomb_set
 
 /obj/item/disk/nuclear/New()

```

Comment:
After the PS is dead, this will be the next big (and annoying) project; killing of *most* instances of `New` in favor of `Initialize`